source/_includes/code_examples/code_example.liquid

```diff
@@ -10,6 +10,12 @@
     {%- if code.css.size -%}
       {%- render 'code_examples/tab', name: 'CSS', syntax: 'css', id: exampleName, enabled: false -%}
     {%- endif -%}
+    {%- if code.scss.size == 1 -%}
+      <a href="/playground#{{ playgroundLinks.scss }}" class="playground-link" target="_blank" data-syntax="scss">Edit on Playground</a>
+    {%- endif -%}
+    {%- if code.sass.size == 1 -%}
+      <a href="/playground#{{ playgroundLinks.sass }}" class="playground-link" target="_blank" data-syntax="sass">Edit on Playground</a>
+    {%- endif -%}
```

Comment:
Note for the designer- Because this is inside the Jquery Tab bar, we have some limitations on markup. If the `<a>` is inside the `<ul>`, it can't be inside a `<li>`, otherwise the link click gets intercepted. I suspect we'll need to move it outside the `ul` as well, but figured I'd defer on the markup.

Reply:
Yeah, I don't think this is worth a ton of complexity. I think it looks fine as is.